client/my-sites/sidebar/sidebar.jsx

```diff
@@ -279,12 +282,19 @@ module.exports = React.createClass( {
 			return null;
 		}
 
+		if ( abtest( 'domainsAddButton' ) === 'button' ) {
+			addDomainButton = <a onClick={ this.onNavigate } href={ addDomainLink } className="add-new">{ this.translate( 'Add' ) }</a>;
+		} else {
+			addDomainButton = '';
```

Comment:
You could also define this empty string as the default on line 259 above and just have an if statement here to override.